test/read.test.ts

```diff
@@ -38,7 +42,13 @@ describe("commands/read", function () {
 
   test("throws with missing file", async function () {
     const consoleError = spy(console, "error");
-    await yargs(["read", "--file", "missing.sql", "--chain", "local-tableland"])
+    await yargs([
+      "read",
+      "--file",
+      "missing.sql",
+      "--baseUrl",
```

Comment:
I included the baseUrl in some of these tests because I wanted to make sure that actual thing under test was the cause of the error without worrying about the order of command validations.